src/main/kotlin/endpoint/RefundServerEndpoint.kt

```diff
@@ -58,10 +55,18 @@ class RefundServerEndpoint(
             })
     }
 
+    /**
+     * Creates a [EthRefundRequest] object from request string
+     */
+    private fun createRequest(txHash: String?): EthRefundRequest? {
+        // TODO replace destination ethereum address
+        return if (txHash != null) EthRefundRequest(txHash) else null
+    }
+
     /**
      * Method return response on stateless invalid request
      */
-    fun onErrorPipelineCall(): String {
+    private fun onErrorPipelineCall(): String {
         logger.error { "Request has been failed" }
         return ""
```

Comment:
I don't think it's a good idea to return empty string

Reply:
fixed